pkg/proxy/proxy.go

```diff
@@ -34,16 +34,18 @@ var (
 // Proxy handler
 type Proxy struct {
 	maxLoadTestsRun int
+	maxListLimit    int64
 	registry        backends.Registry
 	kubeClient      *kube.Client
 }
 
 // NewProxy returns new Proxy handlers
-func NewProxy(maxLoadTestsRun int, registry backends.Registry, kubeClient *kube.Client) *Proxy {
+func NewProxy(maxLoadTestsRun int, registry backends.Registry, kubeClient *kube.Client, maxListLimit int64) *Proxy {
```

Comment:
```suggestion
func NewProxy(maxLoadTestsRun int, maxListLimit int64, registry backends.Registry, kubeClient *kube.Client) *Proxy {
```
what do you think about grouping parameters with same name?

Reply:
I would group them by importance, tbh, but I did not want to change a lot - adding to the end is the simplest case.